### DISCUSSION TITLE
Context Chips in Survey Design: “Okay, but how does it feel?” • Lea Verou

### DISCUSSION THREAD

**Discussion by giscus** [LeaVerou/blog#83] [Blog Comments]
# 
		Context Chips in Survey Design: “Okay, but how does it feel?” • Lea Verou

The story of how a weird little UI to collect sentiment alongside survey responses defied constraints and triumphed over skepticism through usability testing.

https://lea.verou.me/blog/2024/context-chips/

**Comment by DmitrySharabin** [comment]
What a lovely post! So eye-opening, inspiring, and exceptionally well-structured. It's such a good example of how to do the job well, how one sometimes should drop their prejudices and start from scratch, and also how important it is to have an expert in the intersection of fields in a team.

You are an inspiration for so many people, including me. Thank you! ❤️

**Comment by C-Jeril** [comment]
This article delves into the challenges and innovations in survey design, particularly the use of **Context Chips** to collect sentiment data alongside survey responses. The author highlights the difficulty of balancing user engagement with data quality and how usability testing helped refine the design. Context Chips emerged as a powerful solution to convey sentiment quickly and clearly, addressing the inadequacy of traditional survey question formats.

I appreciate the detailed exploration of this design innovation, as it shows how thoughtful UX/UI choices can significantly improve user response rates and data quality. The blend of structured responses and subjective sentiment in a more user-friendly interface shows a clear understanding of the intersection between technology and human experience.

If you're interested in diving deeper into the field of design psychology and how it shapes user experiences, I highly recommend checking out **Kuakua.app**. It offers a wealth of psychological tools and resources, including design psychology insights, to help you better understand human behavior and create more effective, user-centered designs. Whether you're a developer, designer, or someone interested in enhancing your well-being through psychological practices, Kuakua.app provides a unique, science-backed approach to improving your life and work.

Visit **[Kuakua.app](https://kuakua.app/en/docs/product-psychology/the-psychology-of-design)** to explore more!

**Comment by semihcebrail** [comment]
process from developing product "ideas" through survey has transformed into developed survey through survey (: is it illogical to use heat maps to measure dilemma, etc.?
